CHANGELOG.rst

```diff
@@ -78,6 +78,9 @@ v0.7.0 RC2
      occurs when parse_dep attempts to make_mats with a serpent 2 \*_dep.m file
      due to the \*_VOL variable not being a float
 
+*Bug fixes in examples and tutorials
```

Comment:
You'll need a space between * and Bug. I would also suggest we broaden it to just 'fixes' or 'updates' (since everything isn't a bug).